Xamarin.Forms.Core.UnitTests/MultiBindingTests.cs

```diff
@@ -511,13 +512,13 @@ public object[] ConvertBack(object value, Type[] targetTypes, object parameter,
 				if (s == "null" || string.IsNullOrEmpty(s))
 					return null;
 
-				string concatenator = parameter as string ?? " ";
+				string separator = parameter as string ?? " ";
 
-				if (!targetTypes.All(t => t == typeof(string)))
+				if (!targetTypes.All(t=>t==typeof(object)) && !targetTypes.All(t => t == typeof(string)))
 					// Normally we'd return null but throw exception just for unit test to catch
 					throw new Exception("Invalid targetTypes");
```

Comment:
never ever throw in a real world converter. those exception are not, and will never be caught and it'll crash your app. you've been warned